addons/website_livechat/static/src/components/visitor_banner/visitor_banner.scss

```diff
@@ -0,0 +1,90 @@
+// -----------------------------------------------------------------------------
+// Layout
+// -----------------------------------------------------------------------------
+
+.o_VisitorBanner {
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+    display: flex;
+    flex: 0 0 auto;
+    padding: map-get($spacers, 4) map-get($spacers, 2);
+}
+
+.o_VisitorBanner_avatar {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+}
+
+.o_VisitorBanner_avatarContainer {
+    height: 36px;
+    margin-left: map-get($spacers, 1);
+    margin-right: map-get($spacers, 1);
+    position: relative;
+    width: 36px;
+}
+
+.o_VisitorBanner_country {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_history {
+    margin-top: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_historyIcon {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_language {
+    margin-inline-end: map-get($spacers, 3);
+}
+
+.o_VisitorBanner_languageIcon {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_onlineStatusIcon {
+    @include o-position-absolute($bottom: 0, $right: 0);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-flow: column;
+    width: 1.2em;
+    height: 1.2em;
+    line-height: 1.3em;
+    font-size: x-small;
+}
+
+.o_VisitorBanner_sidebar {
+    display: flex;
+    flex: 0 0 50px;
+    justify-content: center;
+    margin-inline-end: map-get($spacers, 2);
+    max-width: 50px;
```

Comment:
Share variable(s)/mixin(s) with `o_Message_sidebar` if the goal is to have the same values.